### ISSUE TITLE
Show rich workspace below recommendatios

### ISSUE THREAD

**Issue by kesselb** [nextcloud/server#18492]
**Is your feature request related to a problem? Please describe.**

![nextcloud test_apps_files__dir=_ fileid=2](https://user-images.githubusercontent.com/3902676/71199205-bcba0300-2295-11ea-9bb0-43fd5cf356ba.png)

I would expect the rich workspace below the recommendations. The rich workspace seems to be related to the current folder while the recommendations are for my whole account. 

**Describe the solution you'd like**

Show rich workspace below recommendations

**Describe alternatives you've considered**

Hide recommendations if rich workspace is enabled

**Comment by kesselb** [comment]
cc @nextcloud/designers

**Comment by juliushaertl** [comment]
Yes, i think that would make sense. cc @jancborchardt Since we were already talking about making it a bit less prominent.

**Comment by jancborchardt** [comment]
> while the recommendations are for my whole account

I’d say that this is only a current shortcoming of recommendations. Ideally they should also be available in subfolders.

Also, the Rich workspace is more related to the breadcrumb part, that is the current folder name, which we could show more presently (as a heading e.g.).

Yes we need to do some more work there, but the order is something we should keep.

**Comment by ZetiX** [comment]
I think it would be great if it's added under the files as if the page loads slow, the folders/files would load first, then the workspace, which pushes the folders/files down and can result in misclicks. This would make it similar to github's layout.
